src/test/Security/encryptionTest.java

```diff
@@ -0,0 +1,82 @@
+package Security;
+
+import Config.MongoConfig;
+import com.google.common.io.Files;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoDatabase;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+
+import static org.junit.Assert.assertEquals;
+
+public class encryptionTest {
+
+  MongoClient client = MongoConfig.getMongoClient();
+  MongoDatabase db = client.getDatabase("staging");
+
+  EncryptionController encryptionController = new EncryptionController(db);
+
+  @Test
+  public void encryptDecryptStringTest() throws GeneralSecurityException, IOException {
+    String string1 = "Hello World 12345 9908";
+    String username = "username";
+
+    String encrypted = encryptionController.encryptString(string1, username);
+
+    String decrypted = encryptionController.decryptString(encrypted, username);
+
+    System.out.println("Encrypting: " + string1);
+    System.out.println("Encrypted Result: " + encrypted);
+    System.out.println("Decrypted Result: " + decrypted);
```

Comment:
remove print lines- all of them
also why are we hardcoding values in- please put them as constants on the top

Reply:
Also can we decrypt large strings?

Reply:
This is a test - shouldn't we hardcode?
We can decrypt large strings locally - size doesn't matter

Reply:
actually hardcoding is fine. It still should be good to write a test for decrypting large strings